msteams-platform/apps-in-teams-meetings/meeting-app-extensibility.md

```diff
@@ -144,12 +141,19 @@ The following image displays the **Contoso** tab with results of poll and feedba
 
 ---
 
-> [!NOTE]
-> Tab layout must be organized when there are more than 10 polls or surveys.
+#### Apps in channel meeting
+
+A public scheduled channel meeting has the same list of apps as its parent team. Installing an app to a channel meeting also makes it available in the parent team, and vice versa.
+
+However, the tab instances in a channel meeting are separate from the tabs in the channel itself. For example, suppose a "Development" channel has a "Polly" tab. If we create a "Standup" meeting in that channel, that meeting would not have a "Polly" tab, until we explicitly [to add tabs to an existing meeting](#to-add-tabs-to-an-existing-meeting).
```

Comment:
> until we explicitly [to add tabs to an existing meeting]

This doesn't read right. "until we explicitly to add tabs" 
Please change the sentence to

> If you create a "Standup" meeting in that channel, that meeting would not have a "Polly" tab, until you explicitly [add the tab to the meeting](#to-add-tabs-to-an-existing-meeting).